core/src/main/java/com/kinandcarta/create/proxytoggle/core/theme/Theme.kt

```diff
@@ -0,0 +1,39 @@
+package com.kinandcarta.create.proxytoggle.core.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+
+private val LightColorPalette = lightColors(
+    primary = VividPurple,
+    surface = LightGray
+)
+
+private val DarkColorPalette = darkColors(
+    primary = Periwinkle,
+    surface = Dark
+)
+
+@Composable
+fun ProxyToggleTheme(
+    darkTheme: Boolean = isSystemInDarkTheme(),
+    isPreview: Boolean = false,
+    content: @Composable () -> Unit
+) {
+    // Temporary fix for previews not working if we use systemUiController...
```

Comment:
What's the issue here?

Reply:
Previews were not rendering: https://github.com/google/accompanist/issues/918
The fix was still in alpha I believe, so I couldn't use it. Removed it now.